great_expectations/data_context/store/html_site_store.py

```diff
@@ -99,6 +99,39 @@ def set(self, key, serialized_value):
         ].set(key.resource_identifier.to_tuple(), serialized_value,
               content_encoding='utf-8', content_type='text/html; charset=utf-8')
 
+    def get_url_for_resource(self, resource_identifier=None):
+        """
+        Return the URL of the HTML document that renders a resource
+        (e.g., an expectation suite or a validation result).
+
+        :param resource_identifier: ExpectationSuiteIdentifier, ValidationResultIdentifier
+                or any other type's identifier. The argument is optional - when
+                not supplied, the method returns the URL of the index page.
+        :return: URL (string)
+        """
+        from great_expectations.data_context.store import FixedLengthTupleFilesystemStoreBackend
```

Comment:
Ditto above re: import style.

Reply:
Yes, the import fails if this import is at the top of the file. Might be circular, but I have not investigated further. Agreed - we need to agree on a "standard procedure" for imports.